src/main/target/STM32F3DISCOVERY/target.h

```diff
@@ -178,6 +178,14 @@
 #define USE_I2C_DEVICE_1
 #define I2C_DEVICE              (I2CDEV_1)
 
+#define LSM303DLHC_I2C                       I2C1
```

Comment:
These will have to go into `src/main/target/CHEBUZZF3/target.h` as well, this target uses the LSM303DLHC as well.

Reply:
My grep-fu was weak, sorry about that